contribution.md

```diff
@@ -8,12 +8,13 @@ All modifications to the repositories and images must be submitted via a Pull Re
 
 ## Dockerfile Guidelines
 
-The images under SCLOrg generally follow [Container Best Practices](http://docs.projectatomic.io/container-best-practices/). Please read through them before submitting a PR.
+The images under SCLOrg generally follow [Container Best Practices](https://docs.docker.com/build/building/best-practices/). Please read through them before submitting a PR.
 
 As for names and labels, [guidelines available here](https://github.com/projectatomic/ContainerApplicationGenericLabels/) are followed.
 
 ### OpenShift
 
+-- FIX URL --
 Ideally, all the images available under sclorg should work fine in the OpenShift environment. For more about what is required from developers, read the [Guidelines for writing Dockerfiles for OpenShift](https://docs.openshift.com/enterprise/3.0/creating_images/guidelines.html).
```

Comment:
Likely: https://docs.redhat.com/en/documentation/openshift_container_platform/3.11/html/creating_images/creating-images-guidelines#overview

Reply:
V novější Openshif dokumentaci tento kus neexistuje.
Je tedy OK ho používat?

Reply:
It does, but it is slightly different: https://docs.redhat.com/en/documentation/openshift_container_platform/4.18/html/images/creating-images